modules/installation-launching-installer.adoc

```diff
@@ -301,22 +301,14 @@ Respond to the installation program prompts.
 For production {product-title} clusters on which you want to perform installation debugging or disaster recovery, select an SSH key that your `ssh-agent` process uses.
 ====
 .. For `Platform`, select `ovirt`.
-.. For `Enter oVirt's API endpoint URL`, enter the URL of the {rh-virtualization} API using this format:
-+
-[source,terminal]
-----
-https://<engine-fqdn>/ovirt-engine/api <1>
-----
-+
-<1> For `<engine-fqdn>`, specify the fully qualified domain name of the {rh-virtualization} environment.
+.. For `Engine FQDN[:PORT]`, enter the fully qualified domain name (FQDN) of the {rh-virtualization} environment.
```

Comment:
".... of the {rh-virtualization} environment" - I think that 'the' word is unnecessary

Reply:
@mgold1234 It's necessary: "the oVirt environment..." or "the Red Hat Virtualization environment..." are both correct.